keps/sig-network/2091-admin-network-policy/README.md

```diff
@@ -402,129 +408,124 @@ type AdminNetworkPolicy struct {
   // Specification of the desired behavior of AdminNetworkPolicy.
   Spec     AdminNetworkPolicySpec
 
-  // ANPStatus is the status to be reported by the implementation, this is not 
-  // standardized in alpha and consumers should report what they see fit in 
+  // ANPStatus is the status to be reported by the implementation, this is not
+  // standardized in alpha and consumers should report what they see fit in
   // relation to their AdminNetworkPolicy implementation
-  // +optional 
+  // +optional
   Status  AdminNetworkPolicyStatus
 }
 
-type AdminNetworkPolicyStatus struct { 
-  conditions   []metav1.Condition  
+type AdminNetworkPolicyStatus struct {
+  Conditions   []metav1.Condition
 }
 
 // AdminNetworkPolicySpec provides the specification of AdminNetworkPolicy
 type AdminNetworkPolicySpec struct {
-  // Priority is an int32 value bound to 0 - 1000, the lowest positive priority, 
-  // "1" corresponds to the lowest importance, while higher priorities have 
-  // higher importance. An ANP with a priority of "0" will be evaluated after all 
+  // Priority is an int32 value bound to 0 - 1000, the lowest positive priority,
+  // "1" corresponds to the lowest importance, while higher priorities have
```

Comment:
"1" corresponds to the highest importance, while higher priorities have lower importance

Reply:
Done